app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,9 @@
         <meta-data
             android:name="com.google.android.geo.API_KEY"
             android:value="@string/google_api_key" />
+        <meta-data
+            android:name="WEATHER_API_KEY"
+            android:value="9ea12af4e349476350e1e525049f75a8" />
```

Comment:
Will the key be hidden? I don't remember where we're supposed to pu keys.

Reply:
If anyone has any ideas I'm open to suggestion. I noticed that our google maps key was there too, so YOLO